source/reference/cutover-process.txt

```diff
@@ -63,19 +88,66 @@ Steps
       migration, you must issue a commit request for each ``mongosync``
       instance.
 
+      Request
+      ~~~~~~~~
+
+      .. literalinclude:: /includes/api/requests/commit.sh
+         :language: shell
+
+      Response
+      ~~~~~~~~
+
+      .. literalinclude:: /includes/api/responses/success.json
+         :language: json
+         :copyable: false
+
       .. note:: 
 
-         Ensure that the ``mongosync`` process response for the
-         ``commit`` request indicates that the ``mongosync`` state is
-         ``COMMITTING`` or ``COMMITTED``, which is the correct
-         ``mongosync`` state when you send a ``commit`` request.
+         After you submit a ``commit`` request, ensure that the ``mongosync`` 
+         state is ``COMMITTING`` or ``COMMITTED``.
```

Comment:
Sorry, I'm still not totally clear on this. How do users determine what state `mongosync` is in? Is that done with the `progress` endpoint? If so, should we clarify that?

Reply:
Yes, they can use the progress endpoint. Fixing now!